src/app/app.component.ts

```diff
@@ -78,10 +82,15 @@ export class AppComponent implements AfterViewInit {
   // Angular hook, after view is initiated
   async ngAfterViewInit(): Promise<void> {
     await this.getAndInitializeRepositoryClientAndServicesAsync();
+    this.lfFieldContainerQueryList.changes.subscribe(async (comps: QueryList<ElementRef<LfFieldContainerComponent>>) => {
```

Comment:
we don't need to do this.. it should exist without doing this when using ViewChildren

Reply:
it does not work without this subscription. at ngAfterViewInit the queryList exist but first is undefined. so we have to call initAsync again when the children become available